src/Twig/Extension/RequirementExtension.php

```diff
@@ -90,15 +99,20 @@ public function groupByCategory($data): array
         return $result;
     }
 
-    public function prepareRequirements(Collection $data): array
+    /**
+     * @return mixed[]
+     */
+    public function prepareRequirements(Collection $collection): array
     {
-        $result = $this->formatVersions($data);
+        $result = $this->formatVersions($collection);
         $result = $this->groupByCategory($result);
-        $result = $this->sortByTitle($result);
 
-        return $result;
+        return $this->sortByTitle($result);
```

Comment:
This is exactly where we want the pipes RFC for PHP 8.1. :smile:   (I don't know if that's going to pass, though.)